Framework/Geometry/src/Objects/MeshObject.cpp

```diff
@@ -101,55 +87,55 @@ bool MeshObject::hasValidShape() const {
 }
 
 /**
-* Determines whether Pt is within the object or on the surface
-* @param Pt :: Point to be tested
+* Determines whether point is within the object or on the surface
+* @param point :: Point to be tested
 * @returns 1 if true and 0 if false
```

Comment:
Comment is wrong. Returns True if within surface, does not return an int, so 0, 1 do not make sense in comment.